### ISSUE TITLE
net.Server is limited to 10 listeners

### ISSUE THREAD

**Issue by hexa00** [theia-ide/theia#410]
Since starting a websocket server adds a listener on the 'upgrade' event of the
http server we need to increase the default 10 listener limit on  net.Server.

It gives erros like:

```
(node:715) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 upgrade listeners added. Use emitter.setMaxListeners() to increase limit
MaxListenersExceededWarning
Possible EventEmitter memory leak detected. 11 upgrade listeners added. Use emitter.setMaxListeners() to increase limit
MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 upgrade listeners added. Use emitter.setMaxListeners() to increase limit
at _addListener (events.js:281:19)
at Server.addListener (events.js:298:10)
at openSocket (/home/eantotr/sync/work/theia/packages/core/lib/node/messaging/connection.js:33:20)
at Object.openJsonRpcSocket (/home/eantotr/sync/work/theia/packages/core/lib/node/messaging/connection.js:22:5)
at _loop_1 (/home/eantotr/sync/work/theia/packages/languages/lib/node/languages-backend-contribution.js:43:20)
at LanguagesBackendContribution.onStart (/home/eantotr/sync/work/theia/packages/languages/lib/node/languages-backend-contribution.js:57:17)
at /home/eantotr/sync/work/theia/packages/core/lib/node/backend-application.js:107:33
at Promise (<anonymous>)
at BackendApplication.start (/home/eantotr/sync/work/theia/packages/core/lib/node/backend-application.js:97:16)
at start (/home/eantotr/sync/work/theia/examples/browser/src-gen/backend/server.js:37:24)
```

**Comment by hexa00** [comment]
Fixed by #409

**Comment by hexa00** [comment]
This is now merged.
